pkg/command/repair/res.yaml

```diff
@@ -19,13 +19,17 @@ intensity: |
   How many token ranges to repair in a single Scylla node at the same time.
   Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
   Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+  Setting intensity to a value greater than max supported results in trimming it internally.
+  See effectively used intensity value in the display of 'sctool progress repair' command.
 
 parallel: |
   The maximum number of Scylla repair jobs that can run at the same time (on different token ranges and replicas).
   Each node can take part in at most one repair at any given moment. By default the maximum possible parallelism is used.
   The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
   The formula to calculate is is as follows: number of nodes / RF.
   For example, the maximum parallelism for a 6 node cluster with RF=3 is 2.
+  Setting parallel to a value greater than max supported results in trimming it internally.
```

Comment:
See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.